math/pronicnumber_test.go

```diff
@@ -0,0 +1,439 @@
+// pronicnumber_test.go
+// description: Returns true if number is pronic, false otherwise
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see pronicnumber.go
+
+package math_test
+
+import (
+	"testing"
+
+	"github.com/TheAlgorithms/Go/math"
+)
+
+func TestPronicNumber(t *testing.T) {
+	var tests = []struct {
+		name          string
```

Comment:
I [still](https://github.com/TheAlgorithms/Go/pull/595#discussion_r1007868973) think that `name` is redundant here. If you insist to have it, I the _rows_ should look like:
- `{"41 is not pronic", 41, false}`,
- `{"42 is pronic", 42, true}`.

Reply:
Fixed this 👍